### PR TITLE
Update droppable config to allow a function

### PR DESCRIPTION
**Pull request by rywilson28**
#3392 
Changes for dynamic droppable

### CODE REVIEW

File: src/utils/Sorter.js [hunk 1]
```diff
@@ -622,13 +622,22 @@ export default Backbone.View.extend({
...
     let droppable = trgModel.get('droppable');
-    droppable = droppable instanceof Backbone.Collection ? 1 : droppable;
-    droppable = droppable instanceof Array ? droppable.join(', ') : droppable;
-    result.dropInfo = droppable;
-    droppable = isString(droppable) ? this.matches(src, droppable) : droppable;
-    droppable =
-      draggable && this.isTextableActive(srcModel, trgModel) ? 1 : droppable;
-    result.droppable = droppable;
+    if (typeof droppable === 'function') {
+      let res = droppable(src, trg);
```
Comment (artf) [inline src/utils/Sorter.js:626]: I'd prefer having `droppable(srcModel, trgModel)` as it's easy to get the DOM from the component (`comp.getEl()`) and it's not the same for the opposite sense

**Review by artf — COMMENTED** [review]
All good, just check a few comments I've added

File: src/utils/Sorter.js [hunk 2]
```diff
@@ -558,7 +558,7 @@ export default Backbone.View.extend({
     const $parent = parent && $(parent);
 
     if (style.overflow && style.overflow !== 'visible') return;
-    if ($el.css('float') !== 'none') return;
+    if ($el.css('float') && $el.css('float') !== 'none') return;
```
[the remark this answers is not in the document]
Comment (rywilson28) [reply src/utils/Sorter.js]: @artf this change was required for the sorter test to pass.  I did make the suggested changes

**Review by artf — APPROVED** [review]

**Comment by artf** [comment]
All good, thanks @rywilson28

**Pull request merged**